index.html

```diff
@@ -42,5 +43,224 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-4">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-5">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+    <div class="sec_card_cnt work-6">
+      <div class="info_card">
+        <div class="cnt_info_card">
+          <h1 class="Title1_card">Profesional Art</h1>
+          <h2 class="Title2_card">Printing Data</h2>
+          <p class="CardDescrip">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard.</p>
+          <ul class="list_feat list_card_gral">
+            <li class="li_btn_2">HTML</li>
+            <li class="li_btn_2">Bootstrap</li>
+            <li class="li_btn_2">Ruby</li>
+          </ul>
+        </div> 
+        <a href="#" class="SeeProjectBtn project_btn_2">See Project</a>
+      </div>
+    </div>
+  </div>
+
+  </section>
+
+  <!-- Section About -->
+  <section class="about back_dark">        
+    <div class="about_container">
+      <h1 class="Tittle_about">About me</h1>
+      <p class="DescripAbout">Hello I'm a software developer! I can help you build a product, feauture or website. Look through some of my work and expererience!. If you like what you see and have a prject you need coded, don't hesitate to contact me.</p>
+      <a class="SeeProjectBtn BtnClass resume_btn" href="">Get My Resume</a>
+    </div>  
+  </section>
+  
+  <!-- Section Skills -->
+  <section class="skills back_dark">
+    <span class="line_div_dark"></span>         
+    <ul>
+      <li class="ul_skill"><h3><img class="ul_skill_icon_d" src="/images/diamond_icon.png" alt="Diamond icon">Language</h3>
```

Comment:
- **[OPTIONAL]** A simple dot could do a big difference in our code, please correct the path of the image by adding a dot at the beginning to avoid broken images.